pandas/tests/window/test_rolling.py

```diff
@@ -1082,28 +1080,133 @@ def test_rolling_sem(frame_or_series):
     tm.assert_series_equal(result, expected)
 
 
-@pytest.mark.xfail(
-    is_platform_arm() or is_platform_power() or is_platform_riscv64(),
-    reason="GH 38921",
-)
 @pytest.mark.parametrize(
-    ("func", "third_value", "values"),
+    ("func", "values", "window", "ddof", "exp_value"),
```

Comment:
At first glance I'm having a hard time deciphering what `exp_value` does - some times it is zero and other times it is "numpy_compute"?